### PR TITLE
[DNS] Docs-in-dash landing page

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
### Summary

PCX-7191

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/dns/get-started.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+For the most common setup, Cloudflare will work as your primary DNS provider. This means that the **DNS** > **Records** page is where you will control which resources are available on the apex domain (`example.com`) or specific subdomains (`blog.example.com`) of your website, as well as control other configurations.
+
+The following links introduce important concepts and guide you through actions you may need to take while having your website on Cloudflare.
+
+- [DNS records](/dns/manage-dns-records/): DNS records communicate information about your domain to visitors and other web services.
+
+- [Nameservers](/dns/nameservers/): In the context of Cloudflare DNS, nameservers refer to authoritative nameservers. When a nameserver is authoritative for `example.com`, it means that DNS resolvers will consider responses from this nameserver when a user tries to access `example.com`.
+
+- [Proxy status](/dns/manage-dns-records/reference/proxied-dns-records/): The proxy status of a DNS record affects how Cloudflare responds to DNS queries to that record.
```
Comment (abstergail) [inline src/content/docs/dns/get-started.mdx]:
```suggestion
- [Proxy status](/dns/manage-dns-records/reference/proxied-dns-records/): Proxy status determines how Cloudflare treats incoming HTTP/S requests to A, AAAA, and CNAME records. When a record is proxied, Cloudflare speeds up and protects HTTP/S traffic with our [cache](/cache/)/[CDN](https://www.cloudflare.com/learning/cdn/what-is-a-cdn/), [DDoS protection](/ddos-protection/), [WAF](/waf/), and [more](/products/?product-group=Application+performance%2CApplication+security).
```

File: src/content/docs/dns/get-started.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - tag: title
+    content: Get started with Cloudflare DNS
+    Description: Review a list of commonly accessed resources to support you throughout your journey as a Cloudflare DNS user.
+---
+
+import { GlossaryDefinition, Render } from "~/components";
+
+Cloudflare's authoritative DNS offering covers a variety of [setups](/dns/zone-setups/).
+
+For the most common setup, Cloudflare will work as your primary DNS provider. This means that the **DNS** > **Records** page is where you will control which resources are available on the apex domain (`example.com`) or specific subdomains (`blog.example.com`) of your website, as well as control other configurations.
```
Comment (abstergail) [inline src/content/docs/dns/get-started.mdx]:
```suggestion
When you [add your domain](/fundamentals/setup/account-setup/add-site/) with Cloudflare as your [primary DNS provider](https://developers.cloudflare.com/dns/zone-setups/full-setup/) (the most common setup), we import your DNS records and become the authoritative DNS provider for that domain, meaning:

- You [manage DNS records](/dns/manage-dns-records/how-to/create-dns-records/) through the Cloudflare dashboard or API. This is how you control which resources are available on the apex domain (`example.com`) or specific subdomains (`blog.example.com`) of your website, as well as control other configurations.
- We [respond to all DNS queries](/fundamentals/concepts/how-cloudflare-works/#how-cloudflare-works-as-a-dns-provider) for your hostnames.
- The DNS records file (AKA the [DNS zone](/dns/concepts/#zone)) is cached across the[ Cloudflare global network](https://www.cloudflare.com/network/), speeding up DNS resolution.
- A, AAAA, and CNAME records are set to [proxy through Cloudflare](/dns/manage-dns-records/reference/proxied-dns-records/) by default.

```

File: src/content/docs/dns/get-started.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The following links introduce important concepts and guide you through actions you may need to take while having your website on Cloudflare.
+
+- [DNS records](/dns/manage-dns-records/): DNS records communicate information about your domain to visitors and other web services.
+
+- [Nameservers](/dns/nameservers/): In the context of Cloudflare DNS, nameservers refer to authoritative nameservers. When a nameserver is authoritative for `example.com`, it means that DNS resolvers will consider responses from this nameserver when a user tries to access `example.com`.
+
+- [Proxy status](/dns/manage-dns-records/reference/proxied-dns-records/): The proxy status of a DNS record affects how Cloudflare responds to DNS queries to that record.
+
+- [How Cloudflare works](/fundamentals/concepts/how-cloudflare-works/): An overview of how Cloudflare works and how you can use it as a DNS provider and as a reverse proxy.
```
Comment (abstergail) [inline src/content/docs/dns/get-started.mdx]:
```suggestion
- [How Cloudflare works](/fundamentals/concepts/how-cloudflare-works/): An overview of how Cloudflare works as a DNS provider and as a reverse proxy.
```

File: src/content/docs/dns/get-started.mdx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- [DNS concepts](/dns/concepts/): Basic concepts organized to help you make sense of specific DNS terminology.
```
Comment (abstergail) [inline src/content/docs/dns/get-started.mdx]:
```suggestion
- [DNS concepts](/dns/concepts/): Fundamental concepts to help you understand DNS terminology.
```

File: src/content/docs/dns/get-started.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    Description: Review a list of commonly accessed resources to support you throughout your journey as a Cloudflare DNS user.
+---
+
+import { GlossaryDefinition, Render } from "~/components";
+
+Cloudflare's authoritative DNS offering covers a variety of [setups](/dns/zone-setups/).
+
+For the most common setup, Cloudflare will work as your primary DNS provider. This means that the **DNS** > **Records** page is where you will control which resources are available on the apex domain (`example.com`) or specific subdomains (`blog.example.com`) of your website, as well as control other configurations.
+
+The following links introduce important concepts and guide you through actions you may need to take while having your website on Cloudflare.
```
Comment (abstergail) [inline src/content/docs/dns/get-started.mdx]:
```suggestion
The following links introduce important concepts and guide you through actions you may need to take while having your website or application on Cloudflare.
```

File: src/content/docs/dns/get-started.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  - tag: title
+    content: Get started with Cloudflare DNS
+    Description: Review a list of commonly accessed resources to support you throughout your journey as a Cloudflare DNS user.
+---
+
+import { GlossaryDefinition, Render } from "~/components";
+
+You can use Cloudflare DNS with a variety of [setups](/dns/zone-setups/).
+
+In the most common setup (full), you [add your domain](/fundamentals/setup/manage-domains/add-site/), import your [DNS records](/dns/manage-dns-records/), and [update your nameservers](/dns/nameservers/update-nameservers/) to make Cloudflare your primary authoritative DNS provider. Once the setup is completed:
```
Comment (pedrosousa) [inline src/content/docs/dns/get-started.mdx:16]: I feel like this page could benefit from some sections dividing the content.

How about the following:
```
You can use Cloudflare DNS with a variety of [setups](/dns/zone-setups/).

## Common setup (full)

In the most common setup (full), you perform the following steps to make Cloudflare your primary authoritative DNS provider:

1. [Add your domain](/fundamentals/setup/manage-domains/add-site/).
2. Import your [DNS records](/dns/manage-dns-records/).
3. [Update your nameservers](/dns/nameservers/update-nameservers/). 

## Additional configuration

Once the setup is completed:

(...)

## More resources

The following links introduce important concepts and will guide you through actions you may need to take while having your website or application on Cloudflare.

(...)
```

File: src/content/docs/dns/manage-dns-records/index.mdx [hunk 7]
```diff
@@ -30,4 +30,11 @@ If you are an Enterprise customer and require more DNS records, contact your acc
...
-<DirectoryListing />
\ No newline at end of file
+### How to
+<DirectoryListing folder="dns/manage-dns-records/how-to" />
+
+### Reference
+<DirectoryListing folder="dns/manage-dns-records/reference" />
+
+### Troubleshooting
+<DirectoryListing folder="dns/manage-dns-records/troubleshooting" />
```
Comment (pedrosousa) [inline src/content/docs/dns/manage-dns-records/index.mdx]: Unless it breaks the formatting somehow, I'd leave an empty line after each heading.

```suggestion
### How to

<DirectoryListing folder="dns/manage-dns-records/how-to" />

### Reference

<DirectoryListing folder="dns/manage-dns-records/reference" />

### Troubleshooting

<DirectoryListing folder="dns/manage-dns-records/troubleshooting" />
```

**Review by pedrosousa — APPROVED** [review]

File: src/content/docs/dns/get-started.mdx (hunk 6 above)
Comment (pedrosousa) [reply src/content/docs/dns/get-started.mdx:16]: I understand that this page will support docs-in-dash readers. If you think we'd have too many headings that would take too much space, feel free to do it differently.

Comment (RebeccaTamachiro) [reply src/content/docs/dns/get-started.mdx:16]: Thank you, Pedro. This was my fear indeed.
But have tried it, checked in with Abby as well, and will add a few headings.

**Pull request merged**